README.md

```diff
@@ -1,67 +1,46 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T1109419)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 [![](https://img.shields.io/badge/💬_Leave_Feedback-feecdd?style=flat-square)](#does-this-example-address-your-development-requirementsobjectives)
 <!-- default badges end -->
 
 # Blazor Grid - Incorporate Drag and Drop Support
 
-The DevExpress Blazor Grid allows you to incorporate drag and drop support for the following usage scenarios:
-
-- Drag rows in a single grid (reorder rows)
-- Drag rows between two grids
-- Drag rows between two grids and drop at a specified position
+The DevExpress Blazor Grid supports drag and drop operations.
 
 ![Drag between two grids](result.png)
 
-The three options listed above are based on [ObservableCollection](https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.observablecollection-1?view=net-6.0) data sources. When used in this manner, the Grid component reacts to changes instantly. Our drag and drop implementation uses three jQuery widgets ([draggable](https://jqueryui.com/draggable/), [droppable](https://jqueryui.com/droppable/), and [sortable](https://jqueryui.com/sortable/)). To begin, add jQuery links to the Layout.csthml file as follows:
-
-```
-<script type="text/javascript" src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
-<script type="text/javascript" src="https://code.jquery.com/ui/1.13.2/jquery-ui.min.js"></script>
-```
+Use the following API members to activate this feature for your Grids:
 
-This GitHub sample also applies [CSS styles](./CS/GridDragAndDrop/wwwroot/css/site.css) to visualize drag and drop operations:
+- [AllowDragRows](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.AllowDragRows) - Specifies whether users can start the row drag operation.
+- [AllowedDropTarget](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.AllowedDropTarget) - Specifies allowed drag and drop targets.
 
-```
-.ui-state-highlight {
-    background-color: rgba(var(--bs-primary-rgb), .3);
-}
+When a user drops rows, the [ItemsDropped](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.ItemsDropped) event fires. In its handler, update the data source: insert rows at the drop position and remove them from the initial position, if required.
 
-.ui-state-hover {
-    background: rgba(var(--bs-primary-rgb), .3);
-}
+## Reorder Rows in a Grid
 
-.ui-draggable-dragging {
-    box-shadow: 0 2px 6px 0 rgb(0 0 0 / 12%);
-}
-```
+If you want to active the reorder capability, set the [AllowDragRows](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.AllowDragRows) property to `true`. You can test this usage scenario on the **Reorder** page.
```

Comment:
I prefer not to use phrases like "the ABC feature/capability". Microsoft never does this in their documentation. They never say they support "the conditional formatting feature". They usually just say "to display X, do Y". 

```suggestion
If you want to let users reorder rows, set the [AllowDragRows](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.AllowDragRows) property to `true`. You can test this usage scenario on the **Reorder** page.
```